src/configs/settings.ts

```diff
@@ -241,3 +257,263 @@ export const symbolLayerConfig = {
   },
   type: 'symbol',
 };
+/** Default table props config */
+export const defaultTableProps = {
```

Comment:
We should not do this in the `settings` module.  Please do this in a helpers or utils module somewhere else.  Reasons why:

1. The settings module should not have any dependencies that are in the codebase
2. We dont want to run into cyclic dependency issues

I'm fine with putting the status column headers in the settings module, but not anything that requires an import from the Reveal web code base.